app/models/media/image.rb

```diff
@@ -0,0 +1,8 @@
+class Media::Image < Media::Object
+  has_attached_file :item, styles: {
+    normal: '720x720>',
+    thumb: '100x100#'
+  }
```

Comment:
I assume this call in addition to the one in the parent class doesn't cause issues? Another option would be to define a `styles` class method that returns the styles hash and then you'd call that from the parent...